R/readwrite.R

```diff
@@ -18,12 +18,17 @@
 #' @export
 #' 
 process_read <- function (handle, pipe = "stdout", timeout = 0)
-{
-  output <- .Call("C_process_read", handle, as.character(pipe), as.integer(timeout))
+{ 
+  output <- .Call("C_process_read", handle, as.character(pipe), 
+                  as.integer(timeout))
   if (!is.character(output)) {
     return(output)
   }
-  
+  while(!identical(output[length(output)], "")){
+    output <- c(output, .Call("C_process_read", handle, as.character(pipe), 
+                              as.integer(timeout)))
```

Comment:
With timeout of zero I was getting no output for the tests (maybe a timing issue). Also I guess the test should be tailored for windows also?